src/main/java/com/google/codeu/data/Marker.java

```diff
@@ -0,0 +1,26 @@
+package com.google.codeu.data;
+
+public class Marker {
+
+  private double lat;
+  private double lng;
```

Comment:
Just my opinion, but I would consider using full forms of the words latitude and longitude, or something a little more clearer as it might be hard for other people reading the code to understand what just lat and lng mean.

Reply:
Same thing with content. Maybe expand a say mapContent or something like that?

Reply:
This is to sync up with the front end google.maps api, if not, the front-end side would be having both lat, latitude and lng, longitude as json fields and variable names which may be even more confusing. Also, the content means the content of the marker instead of the content of the map so it is pretty apt as the variable name in my opinion.